sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { useContext } from "react"
+import { t, GridSection, ViewItem, GridCell } from "@bloom-housing/ui-components"
+import { ListingContext } from "../../ListingContext"
+
+const DetailApplicationAddress = () => {
+  const listing = useContext(ListingContext)
+
+  const getAddressString = (addressType: string): string | undefined => {
+    if (addressType === "leasingAgent") return t("listings.leasingAgentAddress")
+    if (addressType === "mailingAddress") return t("application.contact.mailingAddress")
```

Comment:
Can we use `ListingApplicationPickUpAddressType` enum from the backend-swagger instead of `string`?